ToDoApp/Extention/DateExtenstion.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DateExtenstion.swift
+//  ToDoApp
+//
+//  Created by 최용헌 on 2023/05/23.
+//
+
+import Foundation
+
+// MARK: - 날짜 to String 변경
+extension Date {
+  func convertDateToString(date: Date) -> String {
```

Comment:
```suggestion
  func toString() -> String {
```

`extension`에서 함수를 정의해주는 경우 사용해줄 때
`today.convertDateToString()`와 같이 사용되면 이미 앞에 명사가 붙어있는 거잖아?
그래서 간단하게 `toString()`처럼 함수 이름을 정해주면
`today.toString()`처럼 코드 가독성이 좋아져!